client/packages/core/src/instaql.js

```diff
@@ -258,21 +260,33 @@ function extendObjects(makeVar, store, { etype, level, form }, objects) {
   return Object.entries(objects).map(([eid, parent]) => {
     const childResults = children.map((label) => {
       try {
-        const [nextEtype, nextLevel, join] = makeJoin(
+        const [nextEtype, nextLevel, join, attr, dir] = makeJoin(
           makeVar,
           store,
           etype,
           level,
           label,
           eid,
         );
-        const child = queryOne(store, {
+        const childrenArray = queryOne(store, {
           etype: nextEtype,
           level: nextLevel,
           form: form[label],
           join,
         });
-        return { [label]: child };
+
+        let children = childrenArray;
+        if (store.schema) {
```

Comment:
One thing I realize: 

What if someone _doesn't_ want to use schema yet, but _does_ want this behavior?

I guess we could have another flag. 

Perhaps it's not a problem though 🫡